tests/sentry/features/test_manager.py

```diff
@@ -280,3 +280,13 @@ def test_entity_feature_shim(self):
 
         assert "feat:4" in manager.entity_features
         assert "feat:5" in manager.entity_features
+
+    def test_all(self):
+        manager = features.FeatureManager()
+
+        manager.add("feat:org", features.OrganizationFeature)
+        manager.add("feat:project", features.OrganizationFeature, False)
+        manager.add("feat:system", features.OrganizationFeature, False)
```

Comment:
Need the project/system feature types here

Reply:
yep boo boo